### ISSUE TITLE
About the author section looks too much like post content

### ISSUE THREAD

**Issue by fbender** [potch/hax#9]
There should be a visual indication so it is easier to see where the article ends.

**Comment by potch** [comment]
Yes, this was a concern I wanted to address before launch but ran out of time. I'll likely make the comment section have a different background-color like the light gray of the featured article header to set it out from post content. Thanks for filing!

**Comment by fbender** [comment]
Cool! 

If you’re at it, check out the comments section (incl. form), too. I can’t point a finger on it, but it also seems odd. Maybe a separator or different background color will help.
